File: leetcode/richestCustomerWealth/richestCustomerWealth.py
from typing import List

class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        wealth = sum(accounts[0])
        for i in range(1, len(accounts)):
            curSum = sum(accounts[i])
            if curSum > wealth:
                wealth = curSum
        return wealth

File: leetcode/richestCustomerWealth/richestCustomerWealth.go
package richestcustomerwealth


func maximumWealth(accounts [][]int) int {
	wealth := 0
	for i := 0; i < len(accounts); i++ {
		v := 0
		for j := 0; j < len(accounts[i]); j++ {
			v += accounts[i][j]
		}
		if v > wealth {
			wealth = v
		}
	}
	return wealth
}